Fst2List.cpp

```diff
@@ -40,30 +42,31 @@ namespace unitex {
-          " -o outFile : if this option and -S not exist, save paths at \"file\"lst.txt\r\n"
-          " -[a/t] s/m : mode de automata or transducteur, s=single initial, m = multi-inital\r\n"
+          "Fst2List [-o outFile][-p s/f/d][-a/t s/m] [-m] [-d] [-f s/a] [--io-separator \"Str\"] [--stop-mark \"Str\"][-s \"Str\"] [-r s/l/x \"Str\"] [-l line#] [-i subname]* [-c SS=0xxxx]* fname\n"
+          " fname: input file name with extension \".fst2\"\r\n"
+          " -S: write path on standard output\r\n"
+          " -o outFile: if this option and -S are not used, save paths in \"file\"lst.txt\r\n"
+          " -a/t s/m, --automaton/transducer s/m: a = write outputs, t = ignore outputs, \r\n"
+          "     s=single initial state, m = multiples initals states\r\n"
           "              by default \"-a s\"\r\n"
-          " -l line#  :  max number of line to save[decimal].\r\n"
-          " -i sname : stop call of the exploitation at this sub-graphe \"sname\"\r\n"
-          " -p s/f/d : mode de extrait word, s=each sub graphs, f=full path(default),\r\n"
-          "            d= debugging. default is 'f'\r\n"
+          " -l line#: max number of line to save[decimal].\r\n"
+          " -i sname: stop exploration at subgraph \"sname\"\r\n"
+          " -p s/f/d: exploration mode, s=independently, printing names of called subgraphs, f=recursively(default),\r\n"
+          "            d=debugging. default is 'f'\r\n"
           " -c SS=0xXXXX: change the symbol string between symbols < and >,\"<SS>\" \r\n"
           "                to a unicode character(0xXXXX)\r\n"
-          " -s \"L[,R]\" : use two strings L, R as the separator each item\r\n"
+          " -s \"L[,R]\": use two strings L, R as parentheses to enclose items\r\n"
           "                   default null\r\n"
-          " -s0 \"Str\" : if transductor mode,set \"str\" as the separator between input and out\r\n"
+          " -g/--io-separator \"Str\": if transducer mode, set \"str\" as the separator between input and output\r\n"
           "                   default null\r\n"
-          " -f  a/s :  if the mode is transductor,the format of output line i0i1SOS1(:s) or i0S0i1S1(:a),i0,i1: input, S0,S1:out\r\n"
-          "       default value is \'s\'\r\n"
-          " -ss \"stop\" : set \"str\" as the mark of stop exploitation at \"<stop>\" \r\n"
-          "                    default null\r\n"
-          " -m : mode special for description with alphabet\r\n"
-          " -d : disable loop check: faster execution at the cost of information about loops\r\n"
-          " -v : verbose mode  default null\r\n"
-          " -r[s/l/x] \"L[,R]\"  : present recusive path(c0|...|cn) by Lc0|..|cnR : default null\r\n"
+          " -f a/s: in transducer mode, format of output line: a=i0s0i1s1, s=i0i1s0s1 (i0,i1: input, s0,s1: output)\r\n"
+          "                   default value is \'s\'\r\n"
+          " -q, --stop-mark \"stop\": quit exploration at \"<stop>\" \r\n"
```

Comment:
`-ss,  --stop_mark "stop": set "str" as the mark of stop exploration at "<stop>". The default value is null. Prefer the long option --stop_mark, the short option -ss is being deprecated`

Reply:
I updated `str` with `stop` in the comment!